test/functional/zapwallettxes.py

```diff
@@ -15,9 +15,11 @@
   been zapped.
 """
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import (assert_equal,
-                                 assert_raises_jsonrpc,
-                                 )
+from test_framework.util import (
+    assert_equal,
+    assert_raises_jsonrpc,
+    wait_until,
```

Comment:
should be `from test_framework.mininode import wait_until`

Reply:
fixed